test/test-pages/tumblr/expected-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Minecraft 1.8 - The Bountiful Update - Minecraft 1.8 - The Bountiful Update",
+  "byline": null,
+  "dir": null,
+  "excerpt": "+ Added Granite, Andesite, and Diorite stone blocks, with smooth versions\n+ Added Slime Block\n+ Added Iron Trapdoor\n+ Added Prismarine and Sea Lantern blocks\n+ Added the Ocean Monument\n+ Added Red...",
```

Comment:
Can you file a followup issue so we use the unicode ellipsis character instead of 3 periods when chopping off text like this?